src/io/flutter/view/FlutterViewFactory.java

```diff
@@ -32,9 +31,6 @@ private static void initFlutterView(@NotNull Project project, FlutterViewMessage
 
   @Override
   public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindow toolWindow) {
-    //noinspection CodeBlock2Expr
-    DumbService.getInstance(project).runWhenSmart(() -> {
```

Comment:
One effect of "run when smart" is that at project open, it delays the callback until IntelliJ finished updating the project's index. So that can be useful.